marlowe-playground-client/src/MainFrame.purs

```diff
@@ -339,6 +342,8 @@ handleAction SendResult = do
   updateContractInState contract
   resetContract
   assign _view (Simulation)
+  marloweEditorResize
+  marloweEditorSetTheme
```

Comment:
Is there something to be said for defining an, "update view and fudge about with monaco," function, so these two steps don't get forgotten?

Reply:
ok, moved some things around as well to be neater